satpy/tests/writer_tests/test_cf.py

```diff
@@ -783,18 +778,19 @@ def _gm_matches(gmapping, expected):
         ds = ds_base.copy()
         ds.attrs['area'] = cosmo7
 
-        with mock.patch('satpy.writers.cf_writer.warnings.warn') as warn:
-            res, grid_mapping = area2gridmapping(ds)
-            warn.assert_called()
-            proj_dict = pyresample.geometry.proj4_str_to_dict(res.attrs['grid_proj4'])
-            self.assertEqual(proj_dict['lon_0'], 4.535)
-            self.assertEqual(proj_dict['lat_0'], 46.0)
-            self.assertEqual(proj_dict['o_lon_p'], -5.465)
-            self.assertEqual(proj_dict['o_lat_p'], 90.0)
-            self.assertEqual(proj_dict['proj'], 'ob_tran')
-            self.assertEqual(proj_dict['o_proj'], 'stere')
-            self.assertEqual(proj_dict['ellps'], 'WGS84')
-            self.assertEqual(grid_mapping.attrs['name'], 'proj4')
+        # with mock.patch('satpy.writers.cf_writer.warnings.warn') as warn:
+        grid_mapping = area2gridmapping(ds)
+        # warn.assert_called()
+        # proj_dict = pyresample.geometry.proj4_str_to_dict(res.attrs['grid_proj4'])
+        # self.assertEqual(proj_dict['lon_0'], 4.535)
+        # self.assertEqual(proj_dict['lat_0'], 46.0)
+        # self.assertEqual(proj_dict['o_lon_p'], -5.465)
+        # self.assertEqual(proj_dict['o_lat_p'], 90.0)
+        # self.assertEqual(proj_dict['proj'], 'ob_tran')
+        # self.assertEqual(proj_dict['o_proj'], 'stere')
+        # self.assertEqual(proj_dict['ellps'], 'WGS84')
+        self.assertIn('crs_wkt', grid_mapping.attrs)
```

Comment:
Maybe not the whole string but I'd be comfortable with checking if certain parts are there. Just in case the WKT changes a bit I don't want to have to update these tests every time.